### PR TITLE
Edits to enclave concept and configuration content 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Edits to enclave concept and configuration content

### CODE REVIEW

File: docs/HowTo/Configure/Enclave.md [hunk 1]
```diff
@@ -4,70 +4,81 @@ description: Configuring Tessera enclave
...
+Specify the same keys as the transaction manager configuration. The remote enclaves can use all key types, including
+vaults. When using a vault with the enclave, include the corresponding JAR on the classpath. For example: 
 
 * `/path/to/azure-key-vault-0.9-SNAPSHOT-all.jar`
 * `/path/to/hashicorp-key-vault-0.9-SNAPSHOT-all.jar`
 
-If using the all-in-one Transaction Manager jar, all the relevant files are included, and just the configuration needs to be updated for the TM.
-
-If using the individual "make-your-own" JARs, you will need the "core Transaction Manager" JAR along with the "Enclave clients" JAR, and add them both to the classpath as such: `java -cp /path/to/transactionmanager.jar:/path/to/enclave-client.jar com.quorum.tessera.Launcher -configfile /path/to/config.json`
+If using the all-in-one Transaction Manager jar, all the relevant files are included, and just the
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/Enclave.md:80]: @Krish1979 - is this referring to when you are using a local enclave?  Or when you're using a remote enclave with a key vault?

File: docs/HowTo/Configure/Enclave.md [hunk 2]
```diff
@@ -4,70 +4,81 @@ description: Configuring Tessera enclave
...
+configuration needs to be updated for the TM.
 
-*[TM]: Transaction Manager
+If using the individual "make-your-own" JARs, you will need the "core Transaction Manager" JAR along
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/Enclave.md:83]: @Krish1979 - same question as previous paragraph

File: docs/HowTo/Configure/Enclave.md (hunk 1 above)
Comment (Krish1979) [reply docs/HowTo/Configure/Enclave.md:80]: yes it refers to the local enclave

File: docs/HowTo/Configure/Enclave.md (hunk 2 above)
Comment (Krish1979) [reply docs/HowTo/Configure/Enclave.md:83]: Here it refers to not using the all inclusive 'tessera-app-<version>-app.jar but individual JARs for various components. It refers to including 'tessera-simple-<version>-app.jar' and 'remote' enclave client jar in class path.

File: docs/HowTo/Configure/Enclave.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/HowTo/Configure/Enclave.md:80]: Updated

File: docs/HowTo/Configure/Enclave.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Configure/Enclave.md:83]: Updated

**Review by bgravenorst — APPROVED** [review]
Looks good to me

**Pull request merged**